### PR TITLE
Release 262.0.0

### PR DESCRIPTION
**Pull request by Gudahtt**
Release candidate for all packages, motivated by https://github.com/MetaMask/core/pull/5011.

### CODE REVIEW

**Comment by DDDDDanica** [comment]
Tested locally with `@metamask/remote-feature-flag` build with `yarn webpack --watch` and the previous build issue with `cockatiel` package. ✅

**Review by DDDDDanica — DISMISSED** [review]

File: packages/profile-sync-controller/package.json [hunk 1]
```diff
@@ -102,8 +102,8 @@
   "dependencies": {
     "@metamask/base-controller": "^7.0.2",
     "@metamask/keyring-api": "^10.1.0",
-    "@metamask/keyring-controller": "^19.0.0",
-    "@metamask/network-controller": "^22.0.2",
+    "@metamask/keyring-controller": "^19.0.1",
+    "@metamask/network-controller": "^22.1.0",
```
Comment (Gudahtt) [inline packages/profile-sync-controller/package.json:106]: These should be `devDependencies` instead. Unrelated to the release though, we can fix this after

File: packages/queued-request-controller/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [8.0.1]
+
+### Changed
+
+- Bump `@metamask/controller-utils` from `^11.4.3` to `^11.4.4` ([#5012](https://github.com/MetaMask/core/pull/5012))
+- Bump `@metamask/selected-network-controller` from `^20.0.0` to `^20.0.1` ([#5012](https://github.com/MetaMask/core/pull/5012))
```
Comment (Gudahtt) [inline packages/queued-request-controller/CHANGELOG.md:15]: This is a `devDependency`, so we can omit this:
```suggestion
```

**Review by Gudahtt — COMMENTED** [review]
Great work @mcmire ! Just two minor issues.

**Review by mcmire — APPROVED** [review]

**Pull request merged**